github/ci/prow-deploy/files/jobs/RHsyseng/rhcos-slb/rhcos-slb-presubmits.yaml

```diff
@@ -0,0 +1,35 @@
+---
+presubmits:
+  RHsyseng/rhcos-slb:
+    - name: pull-rhcos-slb-ci-network-tests-main
+      branches:
+        - main
+      annotations:
+        fork-per-release: "true"
+      always_run: false
+      optional: true
+      decorate: true
+      cluster: prow-workloads
+      decoration_config:
+        timeout: 30m
+        grace_period: 5m
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20210706-a5d2c6b
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "4Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "dnf install -y git go make wget qemu qemu-img swtpm npm >/dev/null && npm i -D tap-junit >/dev/null && tests/test-coreos.sh"
```

Comment:
I think it would be better to move all the commands to the `tests/test-coreos.sh` script, this way the job definition is simpler and you can update what is executed in the job without having to open a PR to project-infra, wdyt?

Reply:
I see what you mean, but I want to separate the "test" script that I don't want to run with root, and the pre-requisites installation that do require running with root.

Reply:
Ok sounds good, maybe you could put all this code in a separate `test/setup.sh` script?

Reply:
sure! will change as soon as [this](https://github.com/RHsyseng/rhcos-slb/pull/27) is merged

Reply:
[Done](https://github.com/kubevirt/project-infra/compare/452c696237174ca9d07a4cca424d47e9cf6871da..4cca4bfa591cca8ab00b85b51cf239f28a74df50)